src/Lecture1.hs

```diff
@@ -31,10 +32,14 @@ module Lecture1
     , lowerAndGreater
     ) where
 
+import Data.List
+
 {- | Specify the type signature of the following function. Think about
 its behaviour, possible types for the function arguments and write the
 type signature explicitly.
 -}
+makeSnippet :: Int -> String -> [Char]
+
```

Comment:
That's the correct type signature 👍🏻 
`String` and `[Char]` are just two different names for the same thing. And you can use either of them. Usually, everyone uses `String` .

Also, usually there's no empty line between the type signature and the implementation 👾